domainlab/utils/generate_benchmark_plots.py

```diff
@@ -143,28 +167,31 @@ def scatterplot_matrix(dataframe_in, file=None, reg=True, distinguish_param_setu
     distinguish_param_setups: if True the plot will not only distinguish between models,
         but also between the parameter setups
     '''
-    dataframe = dataframe_in.copy()
+    dataframe = dataframe_in.iloc[:, G_DF_PLOT_COL_START:].copy()
     index = list(range(5, dataframe.shape[1]))
```

Comment:
we need another MACRO for 5 i guess?

Reply:
we also need a comment why it is 5 at the place when the macro is defined, so that in the future, when we want to change column names, we just need to change the MACROS there.

Reply:
I think I found a solution with using the existing macros